src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownyEventListener.java

```diff
@@ -80,14 +81,14 @@ public void onTownyLoadLanguages(TranslationLoadEvent event) {
 			for (Map.Entry<String, String> map : translations.get(language).entrySet())
 				event.addTranslation(language, map.getKey(), map.getValue());
 	}
-	
-    /*
-     * Update town peacefulness counters.
-     */
+
     @EventHandler
     public void onNewDay(PreNewDayEvent event) {
-        if (SiegeWarSettings.getWarSiegeEnabled() && SiegeWarSettings.getWarCommonPeacefulTownsEnabled()) {
-            TownPeacefulnessUtil.updateTownPeacefulnessCounters();
+        if (SiegeWarSettings.getWarSiegeEnabled()) {
+            if(SiegeWarSettings.getWarCommonPeacefulTownsEnabled()) {
+                TownPeacefulnessUtil.updateTownPeacefulnessCounters();
+            }
+            SiegeWarDominationAwardsUtil.grantGlobalDominationAwards();

```

Comment:
Might be worth checking if SiegeWarSettings.isDominationAwardsGlobalEnabled() here, to keep uniformity with line 88.

Reply:
Done in next push